src/@types/model.ts

```diff
@@ -42,9 +42,11 @@ export interface Page {
 }
 
 export interface Path {
+    id?: string;
```

Comment:
Would `clientId` work here to more clearly indicate that this is not coming from the service?